packages/compat/package.json

```
@@ -56,7 +56,7 @@
     "typescript": "^5.4.5"
   },
   "peerDependencies": {
-    "eslint": "^9.10.0"
+    "eslint": "8.40 | 9"
```

Minimum version 8.40 makes sense, but I think there should be a caret before `8.40` to allow installing ESLint versions v8.41.0-v8.57.1, i.e. where the minor version number is higher than 40. See https://github.com/npm/node-semver?tab=readme-ov-file#caret-ranges-123-025-004. The logical or is written with a double pipe (`||`).

```suggestion
    "eslint": "^8.40 || 9"
```